docs/advanced/refs.md

```diff
@@ -16,43 +16,40 @@ For example, suppose we wanted to create a search bar that captured cursor focus
  ]]
 local SearchBar = Roact.Component:extend("SearchBar")
 
+function SearchBar:init()
+	self.textBoxRef = Roact.createRef()
+end
+
 function SearchBar:render()
 	-- Render our icon and text box side by side in a Frame
 	return Roact.createElement("Frame", {
 		Size = UDim2.new(0, 200, 0, 20),
 	}, {
 		SearchIcon = Roact.createElement("ImageButton", {
 			Size = UDim2.new(0, 20, 0, 20),
-			-- Handle click events on the icon
-			[Roact.Event.MouseButton1Click] = function()
 
-				-- If our capture method is defined, trigger it
-				if self.captureTextboxFocus then
-					self.captureTextboxFocus()
-				end
+			-- Handle click events on the icon
+			[Roact.Event.Activated] = function()
+				self.textBoxRef.current:CaptureFocus()
 			end
 		}),
 
 		SearchTextBox = Roact.createElement("TextBox", {
 			Size = UDim2.new(0, 180, 0, 20),
 			Position = UDim2.new(0, 20, 0, 0),
-			-- We use Roact.Ref to get a reference to the underlying object
-			[Roact.Ref] = function(rbx)
 
-				-- Set a callback function to give focus to the TextBox
-				self.captureTextboxFocus = function()
-					rbx:CaptureFocus()
-				end
-			end
+			-- Use Roact.Ref to get a reference to the underlying object
+			[Roact.Ref] = self.textBoxRef
 		}),
 	})
 end
 ```
 When a user clicks on the outer `ImageButton`, the `captureTextboxFocus` callback will be triggered and the `TextBox` instance will get focus as if it had been clicked on directly.
 
-## Refs During Teardown
+## Refs During Unmount
```

Comment:
Good catch!

Reply:
Oddly enough, it also needed the newline afterwards, otherwise it wasn't rendering at all o.O